.github/release-drafter.yml

```diff
@@ -71,7 +71,7 @@ autolabeler:
   # labels to help with changelog grouping
   - label: "release-drafter:dependency"
     title:
-      - '^(?:Bump|(?:chore\(deps\):|chore\(deps-dev\)):)' # Dependency update PR by Dependabot
+      - '^(?:Bump|(?:chore\(deps\)|chore\(deps-dev\)|build\(deps\)|build\(deps-dev\)):)' # Dependency update PR by Dependabot
```

Comment:
Consider simplifying to: `^(?:Bump|chore\(deps\)|chore\(deps-dev\)|build\(deps\)|build\(deps-dev\))`

Reply:
That misses the `:` after the conventional commit prefixes

PS: The previous pattern had a bug...

Reply:
But there is another simpler one, give me a few minutes

Reply:
`/^(?:Bump|(?:chore|build)\((?:deps|deps-dev)\):)\s.*/i` should be simpler and align better with the other implementations